lib/rdf-borsh/src/borsh_writer.rs

```diff
@@ -39,10 +47,16 @@ impl BorshWriter {
 
     #[allow(unused_mut)]
     pub fn finish(mut self) -> Result<()> {
-        let version = 0u8;
+        let magic_number = b"RDFB";
```

Comment:
Define these magic constants (file magic, version, flags) as constants in the module.